components/homme/src/share/prim_driver_base.F90

```diff
@@ -1356,16 +1376,22 @@ subroutine prim_step_flexible(hybrid, elem, nets, nete, dt, tl, hvcoord, compute
                 end do
              else
                 ! Set np1_qdp to -1 to remap dynamics variables but
-                ! not tracers.
-                call vertical_remap(hybrid, elem, hvcoord, dt_remap, tl%np1, -1, nets, nete)
+                ! not tracers
+                call vertical_remap(hybrid, elem, hvcoord, dt_remap, tl%np1, &
```

Comment:
Here you're _not_ in a `single_column` block but you use nets_in, nete_in. This combined with the condition on the Prim_Advec_Tracers_remap call below implies the intention is something like: dynamics are active, but tracer transport is fully prescribed. In that case, you don't need to even modify subroutine prim_step_flexible _at all_. Its whole purpose is to support long SL tracer transport time steps. But if you're not using large-scale advection, then you might as well set dt_tracer_factor=1 <= dt_remap_factor and never even call this function. If you do that, I recommend you check that if single_column is true, then dt_tracer_factor <= dt_remap_factor or else abort.